pareto/tests/test_operational_model.py

```diff
@@ -136,3 +143,51 @@ def test_run_operational_model(build_operational_model):
     # 1.- add mass balance checks
     # 2.- create test marks (unit, component, etc.) (work with Ludovico/Keith)
     # 3.- skip test if solver doesnt exist
+
+
+def test_operational_model_discrete_water_quality_build(build_operational_model):
```

Comment:
Minor comment, but would it make sense to mark this as either a component or unit test?

Reply:
good catch, I agree. Just added !